src/cobalt.rs

```diff
@@ -23,14 +23,16 @@ pub fn build(source: &Path, dest: &Path, layout_str: &str, posts_str: &str) -> i
     // filename -> text content to the layout map
     for entry in walker.filter_map(|e| e.ok()).filter(|e| e.file_type().is_file()) {
```

Comment:
Yeah, seems like there should be a utility function for creating `WalkDir`s with various filtering options.